feature-libs/asm/customer-360/components/sections/asm-customer-section/asm-customer-section.component.ts

```diff
@@ -51,6 +52,8 @@ export class AsmCustomerSectionComponent implements OnDestroy {
   @Output()
   navigate: EventEmitter<UrlCommand> = new EventEmitter();
 
+  componentClass: Type<any>;
+
```

Comment:
[MEDIUM]
Do we _really_ need this? Or it's a leftover?

Reply:
leftover..... thanks